front/js/shared/tracking.es6

```diff
@@ -78,6 +67,44 @@
       });
   }
 
+  // @todo #759:30m Move PublishedGATracker to a separate file.
+
+  class PublishedGATracker {
+    constructor() {
+      this.published = false;
+    }
+
+    purchase(productsData, txData) {
+      const publishOnce = () => {
+          // Publish only once
+          if (this.published) return;
+
+          // Load ecommerce plugin for gaTracker
+          ga('require', 'ecommerce');  // Ignore ESLintBear (block-scoped-var)
+          const tracker = new GATracker(ga, 'ecommerce');  // Ignore ESLintBear (block-scoped-var)
+
+          tracker.purchase(productsData, txData);
```

Comment:
hope data is consistent with gAnalytics api docs. Looks too simple and suspicious =)